stable/msoms/README.md

```diff
@@ -0,0 +1,61 @@
+# Azure Log Analytics Container Monitoring Solution
+
+Azure Log Analytics is a software-as-a-service offering from Microsoft that allows Enterprise IT to manage any hybrid cloud. It offers log analytics, automation, backup and recovery, and security and compliance.  Sign up for a free subscription on [Azure](https://azure.microsoft.com/en-us/free/) or read more about [Azure Log Analytics ](https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-overview)
+
+## Introduction
+
+This chart deploys an OMS sidecar agent as a daemonset on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager. The OMS agent enables rich and real-time analytics for Docker containers. With this solution, you can see which containers are running on your container hosts and what images are running in the containers. You can view detailed audit information showing commands used with containers. And, you can troubleshoot containers by viewing and searching centralized logs without having to remotely view Docker or hosts. You can find containers that may be noisy and consuming excess resources on a host. And, you can view centralized CPU, memory, storage, and network usage and performance information for containers. For more information refer to the [documentation](https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-containers).
+
+## Prerequisites
+
+- Kubernetes 1.6+ with Beta APIs enabled
+- Add the Container Monitoring solution to the [Azure Log Analytics workspace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/microsoft.containersoms?tab=Overview).
+
+## Installing the Chart
+
+```bash
+$ helm install --name omsagent stable/msoms
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `omsagent` deployment:
+
+```bash
+$ helm delete omsagent
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the MSOMS chart and their default values.
+
+| Parameter                  | Description                        | Default                                                    |
+| -----------------------    | ---------------------------------- | ---------------------------------------------------------- |
```

Comment:
Table incomplete

Reply:
@unguiculus What do you mean the table is incomplete?

Reply:
Can you call out all configurable values in values.yaml. I believe there are a few missing.

Reply:
The table is cleaned now. It should only have image tag, secrets (wsid/key), and domain.

Reply:
Done